trk/src/Screens/ClimbDetail/index.js

```diff
@@ -166,7 +209,15 @@ function ClimbDetail(props) {
     }
   };
 
-  if (climbData.set === 'Competition') {
+  if (isLoading) {
+    return (
+      <View style={styles.center}>
+        <Text>Fetching climb information...</Text>
+      </View>
+    );
+  //  the reason i put this here is because we will eventually display name and grade here when we encode it onto the nfc tags 
+  //  right now it means that if there is no wifi, something is shown on screen
+  } else if (climbData.set === 'Competition') {
     return (
```

Comment:
by making the first if statement a check for isLoading, none of the other return statements are read if the climbData has not yet been fetched (this solves the problem if climbData.set is not defined)